temoa/extensions/unit_commitment/core/model.py

```
@@ -0,0 +1,183 @@
+    Binary,
+    BuildAction,
+    Constraint,
+    Integers,
+    NonNegativeReals,
+    Param,
+    PositiveReals,
+    Set,
+    Var,
+)
+
+import temoa.extensions.unit_commitment.components.commitment as commitment
```

Ruff (0.15.21)

[warning] Use `from temoa.extensions.unit_commitment.components import commitment` in lieu of alias

Replace with `from temoa.extensions.unit_commitment.components import commitment`

(PLR0402)